Conrollers/getConroller.js

```diff
@@ -20,24 +21,24 @@ function getapi(req, res, next) {
   res.status(200).send({ msg: "No content found" });
 }
 function getArticles(req, res, next) {
-  getArticlesMdl(req)
+  const { sortBy, order } = req.query;
+
+  getArticlesMdl(req, sortBy, order)
     .then((data) => {
       res.status(200).send(data);
     })
     .catch((err) => {
-      console.log(err);
       next(err);
     });
 }
 function getarticle_id(req, res, next) {
   const { id } = req.params;
 
-  getarticle_idMdl(id)
+  getarticle_idMdl(req)
```

Comment:
do you intend to send the **entire request object** to your model here? seems risky and unnecessary